modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -387,6 +387,14 @@ public void testAddCustomObjectDefinition() throws Exception {
 				true
 			).build());
 
+		long objectDefinitionId = objectDefinition.getObjectDefinitionId();
+
+		AssertUtils.assertFailure(
+			ObjectDefinitionStatusException.class,
+			"The object definition is already published",
+			() -> _objectDefinitionLocalService.publishCustomObjectDefinition(
+				TestPropsValues.getUserId(), objectDefinitionId));
```

Comment:
Hey @carolmariaabb, I can't seem to do it inline since `assertFailure` apparently requires all values passed in it to be final. 
I can also do it like this, instead, as it is done in other case in this same file:

```
ObjectDefinition finalObjectDefinition = objectDefinition;

		AssertUtils.assertFailure(
			ObjectDefinitionStatusException.class,
			"The object definition is already published",
			() -> _objectDefinitionLocalService.publishCustomObjectDefinition(
				TestPropsValues.getUserId(), finalObjectDefinition.getObjectDefinitionId()));
```

Reply:
No need, in this case you can't keep the `long objectDefinitionId`

Reply:
@larissacribeiro No need, in this case you can keep the `long objectDefinitionId`